L1Trigger/Phase2L1ParticleFlow/src/BJetId.cc

```diff
@@ -0,0 +1,92 @@
+#include "L1Trigger/Phase2L1ParticleFlow/interface/BJetId.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include <cmath>
+
+BJetId::BJetId(const std::string &iInput,
+               const std::string &iOutput,
+               const BJetTFCache *cache,
+               const std::string &iWeightFile,
```

Comment:
The `iWeightFile` parameter appears to be unused now
```suggestion
```

Reply:
This has been removed as part of the FileInPath change